### PR TITLE
proxmox inventory plugin: Easy fix

### PR DESCRIPTION
**Pull request by trhr**
##### SUMMARY
Plugin was crashing on this line on Python 3.9.2 deployed on qemu image with debian bullseye. It doesn't crash anymore.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
community.general.proxmox

##### ADDITIONAL INFORMATION
I suspect you'll grasp it.

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)?

**Review by russoz — APPROVED** [review]
Looking good to me

**Comment by Ajpantuso** [comment]
This fixes #2957 for which the same solution was proposed.
It would be nice to understand why template is not a required field in the `qemu.conf` causing the failure, but it can remain a mystery for now. Will approve when changelog fragment is added.

File: changelogs/fragments/3052_proxmox_inventory_plugin.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - proxmox_inventory_plugin - It would be nice to understand why template is not a required field in the qemu.conf causing the failure, but it can remain a mystery for now (https://github.com/ansible-collections/community.general/pull/3052).
```
Comment (Ajpantuso) [inline changelogs/fragments/3052_proxmox_inventory_plugin.yml:2]:
```suggestion
  - proxmox inventory plugin - fixed plugin failure when a ``qemu`` guest has no ``template`` key (https://github.com/ansible-collections/community.general/pull/3052).
```

**Review by Ajpantuso — CHANGES_REQUESTED** [review]
Improved change description for anyone looking for this fix in the changelog.

**Review by Ajpantuso — APPROVED** [review]
LGTM, thanks!

**Comment by felixfontein** [comment]
@trhr thanks for fixing this!
@russoz @Ajpantuso thanks for reviewing!

**Pull request merged**